Http/Controllers/Backend/ProfileController.php

```diff
@@ -0,0 +1,193 @@
+    {
+
+        try {
+            $data = [];
+
+            $data['permission'] = [];
+            $data['rows'] = config('vaahcms.per_page');
+
+            $data['fillable']['except'] = [
+                'uuid',
+                'created_by',
+                'updated_by',
+                'deleted_by',
+            ];
+
+            $model = new User();
+            $fillable = $model->getFillable();
+            $data['fillable']['columns'] = array_diff(
+                $fillable, $data['fillable']['except']
+            );
+
+            foreach ($fillable as $column) {
+                $data['empty_item'][$column] = null;
+            }
+
+            $custom_fields = Setting::query()->where('category','user_setting')
+                ->where('label','custom_fields')->first();
+
+            $data['empty_item']['meta']['custom_fields'] = [];
+
+            if (isset($custom_fields)) {
+                foreach ($custom_fields['value'] as $custom_field) {
+                    $data['empty_item']['meta']['custom_fields'][$custom_field->slug] = null;
+                }
+            }
+
+            $roles_count = Role::all()->count();
+
+            $data['actions'] = [];
+            $data['name_titles'] = vh_name_titles();
+            $data['countries'] = vh_get_country_list();
+            $data['timezones'] = vh_get_timezones();
+            $data['custom_fields'] = $custom_fields;
+            $data['fields'] = User::getUserSettings();
+            $data['totalRole'] = $roles_count;
+            $data['country_code'] = vh_get_country_list();
+            $data['registration_statuses'] = Taxonomy::getTaxonomyByType('registrations');
+            $response['success'] = true;
+            $response['data'] = $data;
+        } catch (\Exception $e) {
+            $response = [];
+            $response['success'] = false;
+
+            if(env('APP_DEBUG')){
+                $response['errors'][] = $e->getMessage();
+                $response['hint'][] = $e->getTrace();
+            } else {
+                $response['errors'][] = 'Something went wrong.';
+            }
+        }
+
+        return response()->json($response);
+    }
+    //----------------------------------------------------------
+    public function getItem(Request $request): JsonResponse
+    {
+
+        try {
+            $data['profile'] = User::query()->find(Auth::user()->id);
+            $data['mfa_methods'] = config('settings.global.mfa_methods');
+            $data['mfa_status'] = config('settings.global.mfa_status');
+
+            $response['success'] = true;
+            $response['data'] = $data;
+        } catch (\Exception $e) {
+            $response = [];
+            $response['success'] = false;
+
+            if(env('APP_DEBUG')){
+                $response['errors'][] = $e->getMessage();
+                $response['hint'][] = $e->getTrace();
+            } else {
+                $response['errors'][] = 'Something went wrong.';
```

Comment:
@we-vikram-wri231 this should come from translation, fix it at all other places too